### ISSUE TITLE
On compiling in windows

### ISSUE THREAD

**Issue by joaquin-fv** [acids-ircam/nn_tilde#9]
Hi, i've been trying to compile in windows, but I have the following issue:
-I've followed the instructions successfully and copied the .dll files (excluding the .lib) next to Max.exe.
-Don't know what to do next.  I can't find any external objects to copy to my externals folder in max8. In the instructions for Macos there is a .mxo that should appear, is there anything similar for windows? Maybe there should have been a mxe64 file?
-Are there any files in nn_tilde\build\frontend\maxmsp that I should copy in my max path?

Any help is appreciated.
Thank you.
Joaquin

**Comment by CarloCattano** [comment]
make sure to copy the .mxe64 files from ```nn_tilde\src\externals``` to /Documents/Max 8/Library ``` or some folder in the max path that max knows about , usually anywhere in Library is fine .
I create a folder for nn in library and then copy the max helpfile as well that will get you started on the same folder if you will
```nn_tilde\src\frontend\maxmsp\nn_tilde```
Appart of whats mentioned in the README that is .
Its not the most intuituve way , since most builds in the like are usually placed in ../build/... but as of now thats the flow

**Comment by CarloCattano** [comment]
Not sure if something might have changed since my last build.
Try to run this from nn root folder
```
Get-ChildItem  -name -recurse *.mxe64
 ```
Don't have the laptop with me to verify the powershell command but easier than searching by hand everywhere.
Those are the external objects needed.
You could try to compile with verbose and see if anything comes up

> -- Configuring done
-- Generating done
-- Build files have been written to: C:/.../nn_tilde/build

Seems to me that you just run the first part of generating the files for preparing compilation.
Did you run
``` cmake --build . --config Release```
After that?

**Comment by ammlyy** [comment]
> Double check the instructions and make sure you run one by one all commands

Yeah I did everything from scratch, probably I did something wrong in the process. Now there's a *.mxe64 object, thanks Carlo:)

**Comment by joaquin-fv** [comment]
Carlo, I followed your suggestions, found the .mxe64 files in the src/externals folder, and it worked. 
Thanks for the quick response.

**Comment by CarloCattano** [comment]
Glad I could help!

**Comment by smithee77** [comment]
Hi all, sorry to reopen this thread but I'm a bit stuck but I think I'm pretty close...
* I've managed to get 3 files mc.mm~.mxe64, mcs.mm~.mxe64, and nn.~.mxe64
* I've copied them into Max 8 Library folder

...and now? I have to admit that I'm newbie to Max...I suppose that I should say this modules where my trained models are? how can make them work? Any hint to move forward is more than welcome! :)

regards
